doc/usage.rst

```diff
@@ -60,80 +60,81 @@ On the bottom left is a `google` icon.
 .. image:: google.png
 
 If you click on the `google` icon, you will be able to see a bigger version
-of the image, allowing to see it in more detail.
+of the image, allowing you to see it in more detail.
 
 On the bottom right is a `check` icon.
 
 .. image:: checked.png
 
-If you click on this `checked` icon, you will select this image to vote on.
+If you click on this `check` icon, the corresponding image will be selected to be voted upon.
 
 As you select candidates, these candidates will disapear from the central
-list of candidates and move to the grey column on the right where are
-presented all the candidates you selected.
+list of candidates and move to the grey column on the right where all the candidates you selected
+are shown.
 
-At the top of this selection, still in the grey column on the right is
+At the top of this selection, within the grey column on the right, is
 displayed the number of candidates you selected versus the maximum number of
 candidates you are allowed to select.
 
-Once you selection is made, simply click on ``Submit Votes`` on the grey
+Once your selection is made, simply click on ``Submit Votes`` on the grey
 column on the right.
 
-You will be brought back to the list of elections and thanks for voting.
+You will be brought back to the list of elections and with a message thanking you for voting.
 
 .. note:: You may vote on as many or as little candidates as you like up
           to the maximum allowed per elections.
           If you select less than the maximum number of candidates you will
           be able to come back on the election page and complete your vote
-          (thus allowing to vote in several times).
+          (thus allowing you to vote several times).
```

Comment:
You are not allowed to vote several time, but you can split the process over multiple time, thus voting ``in`` several time (ie: vote on 3 candidates before going to work and finish in the evening or the day later)